python-sdk/tests/test_assignment_logger.py

```diff
@@ -27,3 +29,23 @@ def test_has_log_assignment():
 
 def test_has_log_bandit_action():
     AssignmentLogger().log_bandit_action({})
+
+
+def test_event_format():
+    event = None
+
+    class MyAssignmentLogger(AssignmentLogger):
+        def log_assignment(self, assignment_event: Dict):
+            nonlocal event
+            event = assignment_event
+
+    client = init("ufc", assignment_logger=MyAssignmentLogger())
+    result = client.get_string_assignment(
+        "regex-flag", "alice", {"email": "alice@example.com"}, "default"
+    )
+    assert result == "partial-example"
+
+    assert event["allocation"] == "partial-example"
```

Comment:
Does this demonstrate that nested elements, like `base` are no longer present?

Reply:
yes